Temporary disable TestNumbaIntegration.test_from_cuda_array_interface*

pbelevich opened the pull request:
see https://github.com/pytorch/pytorch/issues/54418
see https://github.com/pytorch/pytorch/issues/54429

Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#54430 Temporary disable TestNumbaIntegration.test_from_cuda_array_interface***

Differential Revision: [D27232636](https://our.internmc.facebook.com/intern/diff/D27232636)

pbelevich committed:
Temporary disable TestNumbaIntegration.test_from_cuda_array_interface*

[ghstack-poisoned]

mruberry reviewed (approved):
This looks good but point to in the skip mesesages https://github.com/pytorch/pytorch/issues/54418.

pbelevich committed:
Update on "Temporary disable TestNumbaIntegration.test_from_cuda_array_interface*"

see https://github.com/pytorch/pytorch/issues/54429


Differential Revision: [D27232636](https://our.internmc.facebook.com/intern/diff/D27232636)

[ghstack-poisoned]
